integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -396,6 +396,46 @@ tieredstore:
     high: 0.95
     low: 0.7
 
+## Proxy ##
+proxy:
+  enabled: false # Enable this to enable the proxy for REST API
+  count: 1
```

Comment:
do you want to have 1 proxy per master or per worker?

Reply:
Our [docs](https://docs.alluxio.io/os/user/stable/en/api/FS-API.html?q=proxy#rest-api) seem to suggest that they should be co-located with the Alluxio workers. Let's do that for the time being?